.github/workflows/publish.yml

```diff
@@ -0,0 +1,33 @@
+name: Publish gem
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/cache@v2
+      id: cache
+      with:
+        path: ~/local/rubies
+        key: ruby-2.6.5
+    - uses: ruby/setup-ruby@v1.62.0
+      id: ruby
+      with:
+        ruby-version: 2.6.5
+        cache-available: ${{ steps.cache.outputs.cache-hit == 'true' }}
+    - name: Print version
+      run: ruby --version
+    - name: Build
+      run: |
+        gem install bundler
+        bundle install --jobs 4 --retry 3
+    - name: Publish gem
+      run: gem bump && gem tag && gem release
```

Comment:
how `gem bump` works? does that also commit and push the change? how can we specify the version or what part we want to bump (major, minor or patch)?

Reply:
Docs are here: https://github.com/svenfuchs/gem-release#reference

It seems this gem might not be as easy to work with as [auto](https://intuit.github.io/auto/).